common/build/eslint-plugin-fluid/package.json

```
@@ -0,0 +1,38 @@
+{
+	"name": "@fluid-internal/eslint-plugin-fluid",
+	"version": "0.1.0",
+	"description": "Custom ESLint rules for the Fluid Framework",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "common/build/eslint-pluig-fluid/"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"main": "index.js",
+	"scripts": {
+		"build": "npm run build:readme",
+		"build:readme": "markdown-magic --files \"**/*.md\"",
+		"clean": "rimraf --glob nyc",
+		"format": "npm run prettier:fix",
+		"prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
+		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
+		"test": "mocha src/test"
```

Nit: let's try like this, just confirm that it still runs the tests as expected.
```suggestion
		"test": "mocha src/test/**/*.test.*js"
```